CMSgov/easi-app
# Easi 1620 -- Continuation of Replacing Font Awesome Icons to use SVG icons

# EASI-1620

## Changes and Description

Continuation of #1509

- The following Icons were updated:
  - Bookmark Icon
  - Left Chevron
  - Angle Icons for Responsive Tabs
  - Truncated Text
  - Exclamation Circle
  - System Health Icons
- Removed `BookmarkCardIcon` and its files for seperate click features from icon
https://github.com/CMSgov/easi-app/pull/1515/files?file-filters%5B%5D=.tsx&show-deleted-files=true&show-viewed-files=true#diff-34afd145d6f00a5c04b8ca6a918868cb6206f5db3ea4ff66358632aa0bef689fR61-R67
- Updated various Snapshot 

## How to test this change

1. Verify no Font Awesome icons remains
2. Verify Icons are SVGs

## Visual Table of Changes
| Icon | Before | After |
|:--:|:--:|:--:|
| `Bookmark` | ![image](https://user-images.githubusercontent.com/2349702/156832623-0e556d98-6912-46ce-9e05-72f73ff4c6fd.png) | ![image](https://user-images.githubusercontent.com/2349702/156832642-29e8f24d-cbfd-4b19-9537-867ba525bc48.png) |
| `Left Chevron` |![image](https://user-images.githubusercontent.com/2349702/156832800-06da3eac-20bc-4973-af16-8dc2f99f6d92.png) | ![image](https://user-images.githubusercontent.com/2349702/156832709-8ea70cb4-afd6-4361-87ce-1f7165a6fa3a.png) |
| `Angle Icons for Responsive Tabs` | ![image](https://user-images.githubusercontent.com/2349702/156832273-500f69a1-d4c9-495b-b561-232a9474d959.png) | ![image](https://user-images.githubusercontent.com/2349702/156832358-5939e8c1-c2ad-4937-ba78-a4171f9ee19a.png)
| `<TruncatedText />` More| ![image](https://user-images.githubusercontent.com/2349702/156831527-e310eb56-257f-43f1-8761-f26a29a20236.png) | ![image](https://user-images.githubusercontent.com/2349702/156831758-7a485bd1-a0da-4a7b-baee-0453e989a771.png) |
| `<TruncatedText />` Less| ![image](https://user-images.githubusercontent.com/2349702/156831596-49640655-9870-492d-a815-f9a5bcd7a07f.png) | ![image](https://user-images.githubusercontent.com/2349702/156831689-10ef6daf-f1d2-4105-9b1e-404e32d4ee4e.png) |
| `Exclaimation Circle` |![image](https://user-images.githubusercontent.com/2349702/156831124-002b1bbb-a086-473f-8c9c-fdbce8bb5f02.png)| ![image](https://user-images.githubusercontent.com/2349702/156831160-1bc0dcd3-5168-4d6f-8a9b-f2936694e21c.png)|
| `System Health` Success, Warning, Fail | ![image](https://user-images.githubusercontent.com/2349702/156831975-bc0b303b-b8b4-498e-852e-97cc21b42215.png) | ![image](https://user-images.githubusercontent.com/2349702/156831885-ed74d91e-bf5f-421b-94d7-2b117c0a8885.png) |




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
